rust/ql/test/query-tests/security/CWE-022/options.yml

```diff
@@ -0,0 +1,3 @@
+qltest_cargo_check: true
```

Comment:
I don't _think_ this line is needed, it should be inherited from `rust/ql/test/options.yml`?